makefiles/e2e.mk

```diff
@@ -32,20 +32,21 @@ e2e-setup-core-wo-auth:
 
 .PHONY: e2e-setup-core-w-auth
 e2e-setup-core-w-auth:
-	helm upgrade --install                          \
-	    --create-namespace                          \
-	    --namespace vela-system                     \
-	    --set image.pullPolicy=IfNotPresent         \
-	    --set image.repository=vela-core-test       \
-	    --set applicationRevisionLimit=5            \
-	    --set dependCheckWait=10s                   \
-	    --set image.tag=$(GIT_COMMIT)               \
-	    --wait kubevela                             \
-	    ./charts/vela-core                          \
-	    --set authentication.enabled=true           \
-	    --set authentication.withUser=true          \
-	    --set authentication.groupPattern=*         \
-	    --set featureGates.zstdResourceTracker=true
+	helm upgrade --install                              \
+	    --create-namespace                              \
+	    --namespace vela-system                         \
+	    --set image.pullPolicy=IfNotPresent             \
+	    --set image.repository=vela-core-test           \
+	    --set applicationRevisionLimit=5                \
+	    --set dependCheckWait=10s                       \
+	    --set image.tag=$(GIT_COMMIT)                   \
+	    --wait kubevela                                 \
+	    ./charts/vela-core                              \
+	    --set authentication.enabled=true               \
+	    --set authentication.withUser=true              \
+	    --set authentication.groupPattern=*             \
+	    --set featureGates.zstdResourceTracker=true     \
+	    --set featureGates.zstdApplicationRevision=true
```

Comment:
does it mean we don't test uncompressed apprevision any more?

Reply:
There are targets that don't use compressions, so we can say uncompressed apprev is still tested. 

https://github.com/kubevela/kubevela/blob/13029c07767a129c545abc9652ec94b2d2b79433/makefiles/e2e.mk#L21-L31

But yes, if we want to test every combination, we may want some changes to our testing infrastructure. I'm not sure how this can be done elegantly.

Reply:
OK, I think it's fine in this case, I'm just afraid if we haven't test any cases for uncompressed mode.